superset-frontend/src/dashboard/components/DashboardBuilder/DashboardContainer.tsx

```diff
@@ -33,13 +33,9 @@ import { getRootLevelTabIndex } from './utils';
 
 type DashboardContainerProps = {
   topLevelTabs?: LayoutItem;
-  handleChangeTab: (event: SyntheticEvent<TabContainer, Event>) => void;
 };
 
-const DashboardContainer: FC<DashboardContainerProps> = ({
-  topLevelTabs,
-  handleChangeTab,
```

Comment:
`handleChangeTab`  was never called during runtime and the function interfaces were not compatible:
`TabContainer.onSelect` does not provide the data to `(event: SyntheticEvent<TabContainer, Event>) => void`